Python/Product/PythonTools/PythonTools/LanguageServerClient/LanguageServerSettings.cs

```diff
@@ -102,6 +107,34 @@ public class PythonAnalysisSettings {
                 /// Offer auto-import completions.
                 /// </summary>
                 public bool autoImportCompletions;
+
+                /// <summary>
+                /// Index installed third party libraries and user files for language features such as auto-import, add import, workspace symbols and etc.
+                /// </summary>
+                public bool? indexing;
```

Comment:
Public fields in C# should be capitalized (PascalCase). However, in this case they are lowercase because the class is serializable and I assume the case matters to pylance. 

It's OK to leave it how you have it, for consistency with the rest of the file. But just for your own information, it's possible to do both proper C# naming AND control the casing of the serialized json. See https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-customize-properties?pivots=dotnet-6-0#customize-individual-property-names for more information if you are interested. 😄